projects/analytics_lambdas/main.tf

```diff
@@ -0,0 +1,79 @@
+resource "aws_s3_bucket" "bucket" {
+  bucket = "${var.s3_bucket}"
```

Comment:
This bucket has already been created elsewhere, so suspect this will conflict. We may have to just hardcode the `id` when we reference it?